python/arcticdb/version_store/library.py

```diff
@@ -1288,6 +1288,73 @@ def get_description(self, symbol: str, as_of: Optional[AsOf] = None) -> SymbolDe
             date_range=info["date_range"],
         )
 
+    def get_description_batch(
+        self, symbols: List[Union[str, ReadRequest]]
+    ) -> List[SymbolDescription]:
+        """
+        Returns descriptive data for a list of ``symbols``.
+
+        Parameters
+        ----------
+        symbols : List[Union[str, ReadRequest]
+            List of symbols to read.
+            Params columns, date_range and query_builder from ReadRequest are not used
+
+        Returns
+        -------
+        List[SymbolDescription]
+            A list of the descriptive data, whose i-th element corresponds to the i-th element of the ``symbols`` parameter.
+
+        See Also
+        --------
+        SymbolDescription
+            For documentation on each field.
+        """
+        symbol_strings = []
+        as_ofs = []
+        def handle_read_request(s_):
```

Comment:
Minor: it's unusual to have underscores at the end of variable names in Python. C++ code conventions do not apply in Python.

Reply:
done